Signal/src/ViewControllers/DebugUI/DebugUICalling.swift

```diff
@@ -27,26 +27,30 @@ class DebugUICalling: DebugUIPage {
         }
 
         let sectionItems = [
-            OWSTableItem(title:"Send 'hangup' for old call") {
+            OWSTableItem(title: "Send 'hangup' for old call") { [weak self] in
```

Comment:
`OWSTableItems` shouldn't retain a strong reference to self or we'll leak the VC.